### ISSUE TITLE
Requirements for adding Gamajo-Template-Loader into a Distrubuted theme

### ISSUE THREAD

**Issue by blachawk** [GaryJones/Gamajo-Template-Loader#37]
If I'm developing a theme to be distrubuted on the WordPress Theme Directory or Theme Forest, are there any requirements other than adding the given code and LISCENCE file into our packages?

**Comment by GaryJones** [comment]
You are correct - the class file (and optionally LICENSE) is all that's needed, if that's all you want :-)

Thanks for the taking the time to ask 👍
